app/views/admin/statistics_announcement_unpublishings/new.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :page_title, "Unpublish statistics announcement" %>
+<% content_for :context, @statistics_announcement.title %>
+<% content_for :title, "Unpublish statistics announcement" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @statistics_announcement)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">Take announcements off GOV.UK if they:</p>
+
+    <%= render "govuk_publishing_components/components/list", {
+      label: "Take announcements off GOV.UK if they:",
+      visible_counters: true,
+      items: [
+        "are a duplicate of another announcement",
+        "have been published too soon or in error"
+      ]
+    } %>
+
+    <% unless @statistics_announcement.cancelled? %>
+      <%= render "govuk_publishing_components/components/inset_text", {
+        text: sanitize("If the statistics are no longer being released, #{link_to("cancel the release", [:cancel, :admin, @statistics_announcement], class: "govuk-link")}.")
+      } %>
+    <% end %>
+
+    <%= form_for @statistics_announcement, url: admin_statistics_announcement_unpublish_path(@statistics_announcement), method: :post do |form| %>
```

Comment:
super nitty & optional. a form uses `:post` by default so we don'y need to pass it in as a method

Reply:
Tested this and tests failed. Reverted. and so ready for merge :)